components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineSession.kt

```diff
@@ -104,7 +104,11 @@ class SystemEngineSession(
     /**
      * See [EngineSession.reload]
      */
-    override fun reload() {
+    override fun reload(flags: LoadUrlFlags) {
+        if (flags.contains(LoadUrlFlags.BYPASS_CACHE)) {
+            webView.clearCache(true)
```

Comment:
I think it's totally fine to ignore the param for `WebView` for now. `clearCache` isn't just clearing the cache for this session but `Clears the resource cache. Note that the cache is per-application, so this will clear the cache for all WebViews used.`

Reply:
Fixed and added a doc comment